testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -1606,14 +1606,22 @@ fn simple_neon_integration() {
     {
         let prom_http_origin = format!("http://{}", prom_bind);
         let client = reqwest::blocking::Client::new();
-        let res = client
-            .get(&prom_http_origin)
-            .send()
-            .unwrap()
-            .text()
-            .unwrap();
-        let expected_result = format!("stacks_node_stacks_tip_height {}", tip.stacks_block_height);
-        assert!(res.contains(&expected_result));
+        let info = get_chain_info_result(&naka_conf).unwrap();
+        let stacks_tip_height = info.stacks_tip_height;
+        wait_for(10, || {
+            let res = client
+                .get(&prom_http_origin)
+                .send()
+                .unwrap()
+                .text()
+                .unwrap();
+            let expected_result = format!(
+                "stacks_node_stacks_tip_height {}",
+                stacks_tip_height - 1_u64
```

Comment:
Why is this height - 1?

Reply:
I think it has something to do with the relayer and this specific metric not getting updated. Not sure exactly what the cause is, I know @jferrant noticed this too.

Reply:
Yeah I had this same issue. I could not figure out WHY this was happening though. :(  It was updated in only very specific code paths that did not seem to happen here and I couldn't reason out that they SHOULD so I think it is correct to check -1.